docs/docs/releases/0.2.0.md

```diff
@@ -481,137 +482,120 @@ We will assume you're using our provided`docker-compose.yml` file. If you have h
     docker exec -it mathesar_db psql -U mathesar -d mathesar_django
     ```
 
-1. At the `psql` prompt, then run:
-
+1. At the `psql` prompt, run:
     ```sql
-    CREATE DATABASE mathesar_temp;
+    SELECT db_name,host,port FROM mathesar_database;
     ```
+    - The above SQL query will provide the list of connected database names along with their hosts and port.
+    - Copy and paste them in a text file somewhere. We will need them after installing 0.2.0.
 
-1. Log out with `CTRL+D`, and log back into the `mathesar_temp` database:
+1. Now run the following:
 
-    ```bash
-    docker exec -it mathesar_db psql -U mathesar -d mathesar_temp
-    ```
+    ```sql
+    -- Create a temporary database
+    CREATE DATABASE mathesar_temp;
 
-1. Then run:
+    -- Change current database to the temporary database
+    \c mathesar_temp;
 
-    ```sql
+    -- Drop the old mathesar_django database
     DROP DATABASE mathesar_django;
-    CREATE DATABASE mathesar_django OWNER mathesar;  -- or whichever role you use to run Mathesar
+
+    -- Create a new mathesar_django database to use with 0.2.0
+    -- Set owner to whichever role you use to run Mathesar
+    CREATE DATABASE mathesar_django OWNER mathesar;
+
+    -- Change current database to the new mathesar_django database
+    \c mathesar_django;
+
+    -- Drop the temporary database
+    DROP DATABASE mathesar_temp;
     ```
 
-1. Log out with `CTRL+D`, and log into your new `mathesar_django` database:
+1. Log out with `CTRL+D`.
 
+1. Stop Mathesar.
     ```
-    docker exec -it mathesar_db psql -U mathesar -d mathesar_django
+    docker compose -f /etc/mathesar/docker-compose.yml down
```

Comment:
We've avoided changing names for any of the docker compose services specifically so they _don't_ have to stop Mathesar for upgrade steps.

Were you getting errors when upgrading without stopping the old services? If so, what were they?

Reply:
We are instructing the user to do a `docker compose up` later down and I don't think we can assume the user wouldn't face any issues during that process. I felt it is simpler to ask the user to stop and start after new installation.